bin/setenv-docker-customize.sh

```diff
@@ -717,6 +717,55 @@ else
   touch /opt/exo/_done.patches
 fi
 
+# -----------------------------------------------------------------------------
+# Import self-signed certificates to Keystore
+# -----------------------------------------------------------------------------
+if [ -f /opt/exo/_done.ssca ]; then
+  echo "INFO: self-signed certificates keystore import already done! skipping this step."
+else
+  # self-signed certificates authorization
+  if [ -z "${EXO_SELFSIGNEDCERTS_HOSTS:-}" ]; then
```

Comment:
We may need to apply it for more than one host

Reply:
Yes it is, it supports many hosts at the same time, please look at the while instruction below.